images/linux/scripts/installers/aliyun-cli.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+################################################################################
+##  File:  aliyun-cli.sh
+##  Desc:  Installs Alibaba Cloud CLI
+################################################################################
+
+# Source the helpers for use with the script
+source $HELPER_SCRIPTS/document.sh
+
+# Install Alibaba Cloud CLI
+page_URL=$(curl -s https://github.com/aliyun/aliyun-cli/releases/latest | cut -d "\"" -f 2)
```

Comment:
I think depending on the html to find things will break eventually. We should be using the Releases API which will even give you a download URL. see: https://api.github.com/repos/aliyun/aliyun-cli/releases/latest

Reply:
Thanks! Fixed it using Releases API.